core/templates/components/site-search/site-search.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Global Site Searach Component
+ *
+ * A default link component.
+ *
+ * Available variables:
+ * - action: Action of the form field
+ * - method: Method of the form field
+ * - input_attributes: Additional attributes for the input field
+ * - placeholder: Placeholder text for the input field
+ * - button_attributes: Additional button attributes
+ * - search_icon_src: The href property of the icon image
+ * - additional_fields: Any additional fields, e.g., hidden input fields
+ *
+ */
+#}
+<div class="su-site-search" role="search">
+ <form action="{{ action }}" method="{{ method }}" accept-charset="UTF-8">
+  <label class="su-site-search__sr-label" for="search-field">Search this site</label>
+  <input{{ input_attributes }} type="text" id="search-field" class="su-site-search__input" placeholder="{{ placeholder }}" maxlength="128">
```

Comment:
field name?

Reply:
Added - using same variable for search input name and ID, and also its label's "for" attribute. Let me know if you think that's a problem for them to be all the same.